lib/utils/initial-questions.js

```diff
@@ -25,7 +25,10 @@ function inquirerInput(name, message) {
 
 const questions = [
 	inquirerInput('entry','What is the name of the entry point in your application?'),
-	inquirerInput('output','What is the name of the output directory in your application?')
+	inquirerInput('outputPath','What is the name of the output directory in your application?'),
```

Comment:
Missing whitespace. @pksjce I'm wondering if you got the lint-staged as a pre-commit hook in place since it should fix such issues automatically. Or are we missing an eslint rule for that?